### PR TITLE
chore: make maker listen on `0.0.0.0:8000`

### LINKED ISSUE #234: Make rocket listen on 0.0.0.0:8000
Maker should listen on `0.0.0.0:8000`

<details>
<summary>Outdated</summary>

> 
> https://github.com/itchysats/10101/pull/233 builds a docker container but the binary fails at the moment. This could be because of rocket. 
> 
> ```
> Error: Cannot assign requested address (os error 99)
> ```
> 
> To solve this I would make maker listen on 0.0.0.0:8000 instead of `localhost`.

</details>

### PR DESCRIPTION
**Pull request by bonomat**
resolves #234

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]

File: maker/src/main.rs [hunk 1]
```diff
@@ -39,7 +39,11 @@ async fn main() -> Result<()> {
 
     let (_, quote_receiver) = bitmex::subscribe()?;
 
-    let mission_success = rocket::build()
+    let figment = rocket::Config::figment()
+        .merge(("address", "0.0.0.0"))
```
Comment (klochowicz) [inline maker/src/main.rs:43]: perhaps a comment stating that we need this for Docker would be useful - might be useful for next projects :)

**Review by klochowicz — APPROVED** [review]
LGTM!

File: maker/src/main.rs (hunk 1 above)
Comment (bonomat) [reply maker/src/main.rs:43]: I 'll overwrite this when doing  #217 and let the user pass it in.

**Pull request merged**